bazel/external/libcircllhist.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/src/circllhist.c b/src/circllhist.c
+index 8ff1221..62674d8 100644
+--- a/src/circllhist.c
++++ b/src/circllhist.c
+@@ -69,7 +69,11 @@ static const hist_bucket_t hbnan = { (int8_t)0xff, 0 };
+ } while(0)
+ #define ASSERT_GOOD_BUCKET(hb) assert(hist_bucket_is_valid(hb))
+ #else
++#ifdef WIN32
```

Comment:
@postwait any chance of just getting this change upstream so we don't need this patch and can bump SHA?

Reply:
Oops nevermind I see a PR was already opened.